docs/schema/index-management/index-lifecycle.md

```diff
@@ -26,28 +26,72 @@ mgmt.commit();
 
 ## Index states and transitions
 
-![States and transitions](index-lifecycle.png)
+The diagram below shows the possible states of an index in JanusGraph.
+Transitions which are not recommended or should be used with caution are shown in grey.
```

Comment:
I don't like the formulation "are not recommended" very much to be honest. Maybe "are not recommended in production" or something like that?